spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UserAgentHeaderProvider.java

```diff
@@ -29,18 +29,13 @@
  * @author Chengyuan Zhao
  * @author Mike Eltsufin
  */
-public class UsageTrackingHeaderProvider implements HeaderProvider {
-
-	/**
-	 * The hard-coded version string reported with usage info.
-	 */
-	public static final String TRACKING_HEADER_PROJECT_VERSION = "1.1.0.BUILD-SNAPSHOT";
+public class UserAgentHeaderProvider implements HeaderProvider {
```

Comment:
Everything is OK, unless I don't like such a breaking change in the point release. 
Would be great to deprecate in the current version and mention in its Javadocs that it i going to be remove in the next `1.2`.

I don't know how Spring Cloud team manages similar situations, but we in our projects stay with the rule as less breaking changes for end-users as possible.

Reply:
This is really an internal class. Do you think people are relying on it when they modify our auto-configuration? If that's the case, we can just leave the old name. It doesn't matter that much.